Cargo.toml

```diff
@@ -23,6 +23,7 @@ signal-hook = "0.3"
 libc = "0.2"
 pin-project = "1.0"
 regex = "1.10.4"
+async-std = "1.12"
```

Comment:
Is `async-std` necessary?

Reply:
Fixed in the latest commit so that AsyncServer<T> does not depend on a specific async runtime, while it now [uses](https://github.com/tier4/safe_drive/pull/30/commits/43428b95d54732938b57650561108a885536dc87#diff-7cedb00a28a89eaf775680dd92bb0e229d0e344f2c40cf6bee3b26161ab389f5R913) `futures::try_join!` to concurrently process incoming requests.